Formula/gcc.rb

```diff
@@ -81,6 +94,7 @@ def install
       --prefix=#{prefix}
       --libdir=#{lib}/gcc/#{version_suffix}
       --disable-nls
+      --enable-bootstrap
```

Comment:
This is the default, I think

Reply:
This is default before gcc 4.2
Starting from GCC 4.2, to do this you have to configure explicitly with --enable-bootstrap.

Reply:
You should have to unless you are cross-compiling: https://github.com/gcc-mirror/gcc/blob/releases/gcc-11.1.0/configure.ac#L1436-L1439